sources/zsh/history.d/haiti

```diff
@@ -0,0 +1,3 @@
+haiti $HASH
+haiti -e $HASH
+haiti samples $REF
```

Comment:
I could find what this command would do on haiti's repo? Similarly to the change above, would there be any way to make it self-explanatory?

Reply:
I'm added a comment, this still works in the history